python/delta_sharing/reader.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         rest_client: DataSharingRestClient,
         *,
         predicateHints: Optional[Sequence[str]] = None,
-        limitHint: Optional[int] = None
+        limit: Optional[int] = None,
```

Comment:
Are we using `limit` instead of `limitHint` because we will always return fewer than `limit`?

Reply:
Yep